### ISSUE TITLE
Optimize startup time?

### ISSUE THREAD

**Issue by xinhangliu** [brglng/vim-im-select#2]
## Env

* OS : macOS 10.15
* Vim: 8.0.200
* IME: macOS default
  * ABC
  * Pinyin - Simplified

## Startup time profile

```
====================================
Top 10 plugins slowing vim's startup
====================================
1	 26.123   vim-im-select
2	  5.545   coc.nvim
3	  2.976   emmet-vim
4	  2.145   vim-devicons
5	  1.845   vim-gruvbox8
6	  1.739   vim-sneak
7	  1.331   lightline.vim
8	  0.993   fzf.vim
9	  0.930   vim-table-mode
10	  0.850   fzf
====================================
```
**Generated by [hyiltiz/vim-plugins-profile](https://github.com/hyiltiz/vim-plugins-profile)**

## Problem

I like `vim-im-select`. It improves my typing experience greatly. But it takes a relatively long time to start up. Is there any way to cut down the startup time?

**Comment by brglng** [comment]
@xinhangliu Please try latest version and see if the startup time is improved. Thanks.

**Comment by xinhangliu** [comment]
> @xinhangliu Please try latest version and see if the startup time is improved. Thanks.

Thank you for your reply! Still the same. 26 ms on average.

**Comment by brglng** [comment]
@xinhangliu It only takes 1ms in my own test.

```
   vim-tmux-clipboard: (16.649ms) ************************************************************************************************************************
       vim-unimpaired: (15.596ms) ****************************************************************************************************************
             nord-vim: (9.27 ms) ******************************************************************
             coc.nvim: (8.465ms) *************************************************************
          vim-matchup: (8.327ms) ************************************************************
        nerdcommenter: (7.06 ms) **************************************************
     vim-visual-multi: (4.959ms) ***********************************
         vim-polyglot: (4.557ms) ********************************
 vim-textobj-function: (3.772ms) ***************************
            lightline: (3.674ms) **************************
                 gitv: (3.55 ms) *************************
   vim-textobj-indent: (3.544ms) *************************
             fencview: (2.908ms) ********************
vim-textobj-parameter: (2.772ms) *******************
        vim-which-key: (2.569ms) ******************
         vim-devicons: (2.401ms) *****************
         vim-fugitive: (2.181ms) ***************
   vim-textobj-syntax: (1.925ms) *************
    vim-terminal-help: (1.691ms) ************
             asyncrun: (1.478ms) **********
          vim-dirdiff: (1.397ms) **********
        vim-im-select: (1.059ms) *******
           vim-eunuch: (0.99 ms) *******
           asynctasks: (0.957ms) ******
 lightline-bufferline: (0.868ms) ******
         vim-surround: (0.807ms) *****
     editorconfig-vim: (0.645ms) ****
           indentLine: (0.609ms) ****
     vim-textobj-user: (0.599ms) ****
vim-tmux-focus-events: (0.477ms) ***
           vim-sleuth: (0.433ms) ***
           vimspector: (0.418ms) ***
              targets: (0.389ms) **
           vim-cppman: (0.387ms) **
       vim-easy-align: (0.353ms) **
             undotree: (0.273ms) *
          vim-dirvish: (0.263ms) *
             clever-f: (0.204ms) *
             vim-clap: (0.193ms) *
         vim-startify: (0.185ms) *
markdown-preview.nvim: (0.165ms) *
             vinarise: (0.127ms)
         vim-snippets: (0.08 ms)
              vim-ps1: (0.068ms)
      plantuml-syntax: (0.041ms)
            vim-dasht: (0.035ms)
                vista: (0.028ms)
             vim-tmux: (0.017ms)
```

Could you please double check you have updated to the latest version?

**Comment by xinhangliu** [comment]
@brglng Thanks!

It worked after I reinstalled the plugin.

```
====================================
Top 14 plugins slowing vim's startup
====================================
1	  5.474   coc.nvim
2	  2.953   emmet-vim
3	  2.210   vim-devicons
4	  1.762   vim-sneak
5	  1.719   vim-gruvbox8
6	  1.466   lightline.vim
7	  0.989   fzf.vim
8	  0.960   vim-table-mode
9	  0.892   vaffle.vim
10	  0.839   fzf
11	  0.689   fastfold
12	  0.663   vim-im-select
13	  0.663   ultisnips
14	  0.601   vim-surround
====================================
```

**Comment by brglng** [comment]
Fixed by `8effd93f0ad42f5220ed5153e92574ed4a00a7f6`
